tests/test__collection_api.py

```diff
@@ -2464,18 +2464,33 @@ def test__aggregate_lookup_operator_not_string(self):
             str(err.exception))
 
     def test__aggregate_lookup_dot_in_local_field(self):
-        with self.assertRaises(NotImplementedError) as err:
-            self.db.a.aggregate([
-                {'$lookup': {
-                    'from': 'b',
-                    'localField': 'should.fail',
-                    'foreignField': 'arr',
-                    'as': 'b'
-                }}
-            ])
-        self.assertIn(
-            "Although '.' is valid in the 'localField' and 'as' parameters",
-            str(err.exception))
+        self.db.a.insert_many([
```

Comment:
Please add an equivalent test in `test__mongomock` to ensure that the feature is behaving the same as in `pymongo`

Reply:
done